src/hidimstat/ensemble_clustered_inference.py

```diff
@@ -305,31 +325,15 @@ def clustered_inference_pvalue(
         1 - corrected p-values
     """
     # corrected cluster-wise p-values
-    if not group:
-        pval, pval_corr, one_minus_pval, one_minus_pval_corr, cb_min, cb_max = (
-            desparsified_lasso_pvalue(
-                n_samples,
-                beta_hat,
-                theta_hat,
-                precision_diag,
-                **kwargs,
-            )
-        )
-    else:
-        pval, pval_corr, one_minus_pval, one_minus_pval_corr = (
-            desparsified_group_lasso_pvalue(
-                beta_hat, theta_hat, precision_diag, **kwargs
-            )
-        )
 
     # De-grouping
     beta_hat, pval, pval_corr, one_minus_pval, one_minus_pval_corr = _degrouping(
         ward,
-        beta_hat,
-        pval,
-        pval_corr,
-        one_minus_pval,
-        one_minus_pval_corr,
+        desparsified_lassos.importances_,
+        desparsified_lassos.pvalues_,
+        desparsified_lassos.pvalues_corr_,
+        1 - desparsified_lassos.pvalues_,
```

Comment:
you cannot simply replace `one_minus_pval` by `1-desparisifed_lassos.pvalues_`, this will lead to numerical errors. For instance, if i given `desparisifed_lassos.pvalues_` is equal to 1-1.e-9 and is stored as np.float32, its value is 1.0, then  `1-desparisifed_lassos.pvalues_` is 0, which creates some numerical issues later.